docs/gravitino-server-config.md

```diff
@@ -43,14 +43,20 @@ You can also specify filter parameters by setting configuration entries of the f
-| `gravitino.entity.store.kv.deleteAfterTimeMs`     | The maximum time in milliseconds that deleted and old-version data is kept. Set to at least 10 minutes and no longer than 30 days. | `604800000`(7 days)              | No       | 0.3.0         |
+| Configuration item                                | Description                                                                                                                                                                                                                                                       | Default value                    | Required | Since version |
+|---------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------|----------|---------------|
+| `gravitino.entity.store`                          | Which storage implementation to use. Key-value pair storage and Relational storage is currently supported, the default value is `kv`, the optional value is `relational`.                                                                                         | `kv`                             | No       | 0.1.0         |
+| `gravitino.entity.store.kv`                       | Detailed implementation of KV storage. `RocksDB` storage is currently supported, and the implementation is `RocksDBKvBackend`.                                                                                                                                    | `RocksDBKvBackend`               | No       | 0.1.0         |
+| `gravitino.entity.store.relational`               | Detailed implementation of Relational storage. `MySQL` storage is currently supported, and the implementation is `JDBCBackend`.                                                                                                                                   | `JDBCBackend`                    | No       | 0.5.0         |
+| `gravitino.entity.store.relational.jdbcType`      | The database type that the `JDBCBackend` needs to connect to. If you use `MySQL` storage, you should first initialize the database tables yourself by executing the file named `mysql_init.sql` in the `src/main/resources/mysql` directory of the `core` module. | `mysql`                          | No       | 0.5.0         |
+| `gravitino.entity.store.relational.jdbcUrl`       | The database url that the `JDBCBackend` needs to connect to.                                                                                                                                                                                                      | (none)                           | Yes      | 0.5.0         |
```

Comment:
You can refer to https://github.com/datastrato/gravitino/blob/main/docs/security.md.
Yes if you use JdbcBackend.

Reply:
Done